Module/client/src/components/app.jsx

```diff
@@ -1,15 +1,32 @@
 import React from 'react';
+import axios from 'axios';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      // refers to the id number typed in the url
+      id: window.location.href.match(/id\s*=\s*(.*)/)[1] - 1,
+    };
+  }
+
+  componentDidMount() {
+    this.getData();
+  }
+
+  getData() {
+    axios.get('/listinginfo').then((response) => {
```

Comment:
oooo interesting....will check out axios! I used fetch but it's good to check out others too (: